androidsdk/src/test/java/com/wootric/androidsdk/objects/SettingsTest.java

```diff
@@ -127,7 +127,7 @@ public void delegatesMethodsToLocalizedTexts() {
         Settings settings = new Settings();
```

Comment:
I think it would make sense to test CES/CSAT in here as well, what do you think ?